cvmfs/magic_xattr.cc

```diff
@@ -222,28 +227,75 @@ std::string CatalogCountersMagicXattr::GetValue() {
   return res;
 }
 
-bool ChunkListMagicXattr::PrepareValueFenced() {
-  chunk_list_ = "hash,offset,size\n";
+bool ChunkListMagicXattr::PrepareValueFenced(int32_t attr_page_num) {
+  chunk_list_ = "ERROR: No chunks! ";
```

Comment:
Why not keeping the initial value being the header line. The way, we don't need to replace it with the header further down.